vtex-apps/store-form
# Feature/graphql mutations error handling

#### What does this PR do? \*

This PR implements the use of the `createDocumentV2` mutation to submit the form data to Master Data. It also implements error handling arrising from the use of this mutation, such as internal server errors or user input errors, and indicates these errors to the user. Finally it implements a new block `form-success` for indicating to the user when the form is successfully submitted.

#### How to test it? \*

Just go to: https://helena--storecomponents.myvtex.com/
To test the error messages just try submitting the form without anything selected.
To test the submitting success message just fix the errors indicated above and submit again(one thing, due to Styleguide limitations I can't indicate an error with the checkbox input, so be sure to check that too)

#### Describe alternatives you've considered, if any. \*



#### Related to / Depends on \*

https://github.com/vtex-apps/store-graphql/pull/447
